src/util/profile/profile.ts

```diff
@@ -25,14 +25,15 @@ export interface UpdatableProfile {
 }
 
 export interface Profile extends UpdatableProfile {
-  readonly accessToken: Promise<TokenValueType>;
+  readonly accessToken: Promise<string>;
```

Comment:
Don't change this - the type is here so that when the token type changes in the future we don't need to change a ton of code. And it will change - it'll need to add an expiration time eventually, for example.